app/services/api/v3/dashboards/parametrised_charts/flow_values_charts.rb

```diff
@@ -60,6 +60,14 @@ def nodes
             )
           end
 
+          def comparison_view?
+            nodes.size > 1
+          end
+
+          def get_correct_chart_type(default_chart_type)
```

Comment:
I think this method should be taking the node type as parameter, and the comparison_view? check must only take into account nodes of the same node type per chart, and not globally (e.g. if 2 municipalities are selected, the "monicipality node type view" is a comparison view, but the "exporter node type view" is not

Reply:
just to double-check, when you say `selected` you mean, predefined by backend and not selected on the client? so, when we've got more than 1 records in here, i.e.: 
![image](https://user-images.githubusercontent.com/6136899/61212792-29374180-a6fb-11e9-9f82-b62b27b01f84.png)
this is a comparison view?
Also, I noticed I'm returning a `BAR_CHART` type for these cases, to make sure: shouldn't it be a `RANKING_CHART`?